src/Raven.Server/Documents/Indexes/Static/Counters/CounterItemFilterBehavior.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using Sparrow.Platform;
+
+namespace Raven.Server.Documents.Indexes.Static.Counters
+{
+    public class CounterItemFilterBehavior : IIndexItemFilterBehavior
```

Comment:
Can you please put some comment explaining why do we have it? Something that it's related to how counters are stored, that we have counter groups and implications for indexing because of that

Reply:
done